conf/messages.cy

```diff
@@ -19,10 +19,10 @@ feedback.after=yn ein helpu i’w wella.
 # Cash CSV
 cf.cash-account.csv.date=Dyddiad y trafodyn
 cf.cash-account.csv.transactionType=Trafodyn
-cf.cash-account.csv.movementReferenceNumber=MRN y datganiad
-cf.cash-account.csv.uniqueConsignmentReference=UCR y datganiad
-cf.cash-account.csv.declarantEori=Eori y datganydd
-cf.cash-account.csv.importerEori=EORI y mewnforiwr
+cf.cash-account.csv.movementReferenceNumber=<abbr title="Cyfeirnod Symud (MRN)">MRN</abbr> y datganiad
+cf.cash-account.csv.uniqueConsignmentReference=<abbr title="Cyfeirnod Unigryw y Llwyth (UCR)">UCR</abbr> y datganiad
+cf.cash-account.csv.declarantEori=<abbr title="Economic Operators Registration and Identification number">Eori</abbr> y datganydd
```

Comment:
It looks like we should be using **'Rhif Cofrestru ac Adnabod Gweithredwyr Economaidd (EORI)'** as a title. I'd recommend double-checking this. The same applies for line 25

Also, could you please verify whether it should be 'EORI' or 'Eori'? I noticed it's written as 'EORI' elsewhere.

Reply:
@Neozxc yh eori should be in caps. just fixed it